### PR TITLE
Feat/write copy

### PR DESCRIPTION
**Pull request by dshelters**

### CODE REVIEW

File: client/components/homepage/Features.jsx [hunk 1]
```diff
@@ -25,7 +27,8 @@ const Features = () => (
           <div className="col-md-6">
             <div className="icon brand-terciary"><i className="glyphicon glyphicon-facetime-video" /></div>
             <h2>Video & Voice Chat</h2>
-            <p>Enter description for this feature</p>
+            <p>Nothing beats talking a problem over face to face.</p>
+            <p>With Peer to Peer Video Chat, you can talk with your coding partner from anywhere.</p>
```
Comment (misteryeo) [inline client/components/homepage/Features.jsx]: "peer-to-peer video chat"? :)

Comment (jefQuery) [inline client/components/homepage/Features.jsx]: Line 30 is a bit awkward. Try "Nothing beats talking about a problem face to face."
And I don't think Peer to Peer Video Chat needs capitalization

File: client/components/homepage/Features.jsx [hunk 2]
```diff
@@ -36,7 +39,8 @@ const Features = () => (
           <div className="col-md-6">
             <div className="icon"><i className="glyphicon glyphicon-comment" /></div>
             <h2>Text Chat</h2>
-            <p>Enter description for this feature</p>
+            <p>Somethings just need to be written down.</p>
```
Comment (jefQuery) [inline client/components/homepage/Features.jsx]: "Some things" over Somethings.
Mention links: "You don't want to spell out that link over voice"

**Comment by jefQuery** [comment]
Moved to #94

**Pull request closed without merging**